srcpkgs/radicale/template

```diff
@@ -1,34 +1,43 @@
 # Template file for 'radicale'
 pkgname=radicale
-version=1.1.6
-revision=4
+version=3.0.3
+revision=1
 archs=noarch
 wrksrc="Radicale-${version}"
 build_style=python3-module
-pycompile_module="radicale"
 hostmakedepends="python3 python3-setuptools"
-depends="python3-pam python3-requests python3-dulwich"
+depends="python3-pam python3-requests python3-dulwich python3-vobject
+ python3-dateutil python3-passlib python3-bcrypt python3-defusedxml
+ apache-htpasswd"
 short_desc="Complete calendar and contact storing and manipulating solution"
-maintainer="lemmi <lemmi@nerd2nerd.org>"
+maintainer="projectmoon <projectmoon@agnos.is>"
 license="GPL-3.0-or-later"
 homepage="https://radicale.org"
-distfiles="${PYPI_SITE}/R/Radicale/Radicale-${version}.tar.gz"
-checksum=c007198ea45ef797344672c681d4c13f8b4aa85c15c41a1156225767a405c92b
-
+distfiles="https://github.com/Kozea/Radicale/archive/${version}.tar.gz"
+checksum=9e22273dda13938b44e0555a96f63e0491e42c1e58a290864ecae1181580be8c
+provides="radicale-${version}_${revision}"
 conf_files="
  /etc/radicale/config
  /etc/sv/radicale/log/run"
 make_dirs="
  /etc/radicale 755 root root
  /usr/share/radicale/ 755 root root
+ /usr/share/examples/radicale/ 755 root root
  /var/log/radicale/ 700 root root
  /var/lib/radicale 750 radicale radicale"
 system_accounts="radicale"
 radicale_homedir="/var/lib/radicale"
 
 post_install() {
 	vinstall config 644 etc/radicale/
+	vinstall config 644 usr/share/examples/radicale
```

Comment:
If you install a file into the directory, you don't need to to add it to `make_dirs`.